openassessment/xblock/workflow_mixin.py

```diff
@@ -110,21 +112,27 @@ def update_workflow_status(self, submission_uuid=None):
             requirements = self.workflow_requirements()
             workflow_api.update_from_assessments(submission_uuid, requirements)
 
-    def get_workflow_info(self):
+    def get_workflow_info(self, submission_uuid=None):
         """
         Retrieve a description of the student's progress in a workflow.
         Note that this *may* update the workflow status if it's changed.
 
+        Keyword Arguments:
+            submission_uuid (str): The submission associated with the workflow to return.
+                Defaults to the submission created by the current student.
+
         Returns:
             dict
 
         Raises:
             AssessmentWorkflowError
         """
-        if not self.submission_uuid:
-            return {}
+        if not submission_uuid:
```

Comment:
I know that this isn't your code originally, but I'd probably recommend checking for `is not None` explicitly here.